packages/client/libclient-py/quickmpc/qmpc_server.py

```diff
@@ -147,7 +147,7 @@ def __stream_result(stream: Iterable, job_uuid: str, party: int,
         is_ok: bool = True
         res_list = []
         for res in stream:
-            if path is not None:
+            if path and res.status == JobStatus.Value('COMPLETED'):
```

Comment:
Empty files were sometimes generated even when the status was not COMPLETED, resulting in unexpected behavior.